starters/express-typeorm-postgres/README.md

```diff
@@ -118,16 +119,16 @@ In the `src/db/run-seeders.ts` file, we provide a script to seed the database wi
 In order to seed the database, you need to do the following steps:
 
 1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
-2. run `npm run infrastructure:start`
+2. run `npm run infrastructure:up`
 3. run `npm run db:seed`
 
 ### Reset infrastructure
 
-If you for some reason need to clear the contents of your database and you want to reinitialise it, delete the `misc/pg_data` folder and delete the postgres docker container. After that the next `infrastructure:start` command will start up as it would the first time.
+If you for some reason need to clear the contents of your database and you want to reinitialise it, run `infrastructure:down`. After that, the next `infrastructure:up` command will start up as it would the first time.
 
-If you would like to clear your redis cache and reinitialise it, delete the `misc/cache_conf` and the `misc/cache_data` folders and delete the cache docker container.
+If you would like to clear your Redis cache and reinitialise it, delete the `misc/cache_conf` and the `misc/cache_data` folders and delete the cache docker container.
 
-If you would like to clear your redis queue and reinitialise it, delete the `misc/queue_conf` and the `misc/queue_data` folders and delete the queue docker container.
+If you would like to clear your Redis queue and reinitialise it, delete the `misc/queue_conf` and the `misc/queue_data` folders and delete the queue docker container.
```

Comment:
It might be better just to drop these. It is possible to restart the container and attach a new volume, but if you're doing that you are competent enough to do all the Docker stuff yourself.

Our advice should be "want to clear anything out? teardown and restart (`down` followed by `up`) 

```suggestion
```